packages/web-client/app/components/card-pay/layer-one-connect-card/index.ts

```diff
@@ -36,24 +36,23 @@ class CardPayDepositWorkflowConnectLayer1Component extends Component<CardPayDepo
     super(owner, args);
     if (this.hasAccount) {
       next(this, () => {
-        this.currentWalletProviderId =
-          this.layer1Network.strategy.currentProviderId || '';
         this.args.onComplete?.();
       });
     }
   }
-  get currentWalletProvider(): WalletProvider | undefined {
+  get connectedWalletProvider(): WalletProvider | undefined {
```

Comment:
When writing the test for this I realised that the `LayerOneConnectCard` component  could get its provider from the network's strategy directly, so it's now separated from the selection state.

Reply:
Tangential benefit of writing tests!